README.md

```diff
@@ -36,15 +38,33 @@ docker run -p 80:8080 \
 cnwan_adaptor
 ```
 
+## Quickstart
+If you want a minimal working setup, the script ```examples/setup_kubecon_demo.sh``` sets everything up a for you in the adaptor and your SD-WAN controller. Before running it, please:
 
+* Install the bash utility ```jq```
+* Edit the file ```examples/policies_definition.json``` with the tunnels, vpns and sites of your deployment. You can add as many polcies and SLAs as you need. In addition, specify the metadata keys and values used by the CNWAN reader.
 
 ## SDWAN controller configuration
 
-The adaptor makes the following assumptions regarding the controller configuration:
 
-1. The ```policyName``` variable in the ```mappings``` schema corresponds to a previously defined policy in the controller, containing its tunnel or SLA.
-2. The ```sdwanMergedPolicyName``` variable in the ```credentials``` schema  corresponds to another traffic policy that will be used to merge all the policies into a single one, so it can be activated.
-3. The mapping of ```metadataValue``` to ```policyName``` is 1:1 (two metadata values cannot share the same ```policyName```). On the other hand, a single ```metadataKey``` supports any number of ```metadataValue```.
+
+The adaptor supports both vManage Application Aware Routing (SLAs) and Traffic Data (send traffic to a specific tunnel color) policies.
+
+The adaptor requires the following configuration in vManage (the script ```examples/setup_kubecon_demo.sh``` automates this process):
+
+1. Switching vSmart to vManage mode
+2. Creating two empty policies with the same name, one in the Application Aware Routing and other in the Traffic Data sections of Traffic Policy section (in the Custom Options section).
```

Comment:
I would link here for both types of policy into the Viptela documentation, so the user can easily find more info on the topic without searching the web.

Reply:
Added appropriate link.